htdocs/web_portal/img/README.md

```diff
@@ -1,22 +1,25 @@
 | Icon image name | Link to source | License info | Author information|
 |-|-|-|-|
-|tick.png|[Link](http://www.softicons.com/system-icons/crystal-clear-icons-by-everaldo-coelho/actions-agt-action-success-icon)|GNU Lesser General Public License |Everaldo Coelho|
 |add.png|[Link](http://www.softicons.com/system-icons/crystal-project-icons-by-everaldo-coelho/actions-edit-add-icon)|GNU Lesser General Public License |Everaldo Coelho|
 |Clock.png|[Link](https://findicons.com/icon/254545/history)|GNU Lesser General Public License|Alessandro Rei|
+|down_arrow.png|[Link](http://www.softicons.com/toolbar-icons/marmalade-icons-by-icojam/arrow-down-icon)|Freeware commercial license|	Icojam|
 |Enter/Next.png|[Link](http://www.iconarchive.com/show/oxygen-icons-by-oxygen-icons.org/Actions-arrow-right-icon.html)|GNU Lesser General Public License|[Link](http://www.iconarchive.com/icons/oxygen-icons.org/oxygen/authors.txt)|
 |First/Last.png|[Link](http://www.iconarchive.com/show/oxygen-icons-by-oxygen-icons.org/Actions-arrow-right-double-icon.html)|GNU Lesser General Public License|[Link](http://www.iconarchive.com/icons/oxygen-icons.org/oxygen/authors.txt)|
-|user.png|[Link](http://www.iconarchive.com/show/ravenna-3d-icons-by-double-j-design/Users-icon.html)|CC Attribution 4.0|Double-J Design|
-|search.png|[Link](http://www.softicons.com/business-icons/business-icons-by-petalart/search-icon)|CC Attribution-NoDerivs 3.0 Unported|	PetalArt|
-|project.png|[Link](http://www.icons101.com/icon/id_51397/setid_1789/Free_3D_Printer_by_AhaSoft/Project)|CC Attribution|Aha-Soft (website)|
 |home.png|[Link](http://iconbug.com/detail/icon/2884/blue-home-folder/)|CC Attribution 3.0 Unported|[Link](http://www.iconarchive.com/icons/oxygen-icons.org/oxygen/authors.txt)|
-|star.png|[Link](http://www.softicons.com/web-icons/web-2-icon-set-by-anders-bjarnle/star-icon)|Free Use of icon|Anders Bjarnle|
+|key.png|[Link](http://www.softicons.com/toolbar-icons/build-icons-by-design-kindle/key-icon)|Free for commercial use|	Design Kindle|
+|network.png|[Link](https://findicons.com/icon/169709/globe)|GNU Lesser General Public License|YellowIcon|
+|new_window.png|[Link](http://www.softicons.com/toolbar-icons/cyan-bitcons-icons-by-some-random-dude/new-window-icon)|CC Attribution-ShareAlike 3.0 United States|Some Random Dude (<-
+actually the author information)|
```

Comment:
I think this should be one line? It looks like the line break messes up the markdown parsing.

Reply:
Good call.